test/WormholeL2GovernorMetadata.t.sol

```diff
@@ -73,7 +73,7 @@ contract ReceiveWormholeMessages is L2GovernorMetadataTest {
       new bytes[](0),
       MOCK_WORMHOLE_SERIALIZED_ADDRESS,
       L1_CHAIN.wormholeChainId,
-      bytes32("")
+      bytes32("0x1")
```

Comment:
Why do we need this change?

Reply:
That value is the delivery hash and we already have an empty deliveryHash on line 66.